### DISCUSSION TITLE
How to Find Out What Things I Haven't Tested Yet in my Contract with Forge Coverage?

### DISCUSSION THREAD

**Discussion by justAWanderKid** [Cyfrin/foundry-full-course-cu#1772] [General]
i remember in one of the lessons there was a "forge coverage" command plus with an option that creates an file and shows what things We Haven't Tested yet. What was it?

thanks!

**Comment by urosognjenovic** [comment]
I think it was:

```
forge coverage --report debug > file.txt
```

**Comment by EngrPips** [comment]
Hello @tornado06, Please let us know if @thebraveman17 answer is what you are looking for.
